desktop/package.json

```diff
@@ -39,17 +39,17 @@
     "getenv": "0.6.0",
     "immutable": "3.8.1",
     "jsonfile": "2.3.1",
-    "lodash": "4.13.1",
-    "marked": "0.3.5",
-    "material-ui": "0.15.2",
-    "menubar": "5.0.0",
+    "lodash": "4.14.2",
+    "marked": "0.3.6",
+    "material-ui": "0.15.3",
+    "menubar": "5.1.0",
```

Comment:
The icon is highlighting on selection (yay!) but it's not inverting color on highlight like the other icons in my menubar. Do we have a ticket for this already?
<img width="204" alt="screen shot 2016-08-10 at 9 36 49 am" src="https://cloud.githubusercontent.com/assets/1152104/17562108/18f03560-5ede-11e6-8f65-873357660db7.png">